build.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineBuildConfig } from 'unbuild';
```

Comment:
This shouldn't be needed when using module-builder

Reply:
TypeScript says otherwise, it's not that huge of an overhead anyway